deploy/kuberhealthy.yaml

```diff
@@ -211,13 +215,13 @@ spec:
               fieldRef:
                 fieldPath: metadata.namespace
           - name: DAEMON_SET_CHECK
-            value: "true"
+            value: "false"
           - name: DNS_STATUS_CHECK
-            value: "true"
+            value: "false"
           - name: POD_RESTARTS_CHECK
-            value: "true"
+            value: "false"
           - name: POD_STATUS_CHECK
-            value: "true"
+            value: "false"
```

Comment:
we should set these back to `true` and only keep them `false` in our testing environment.